src/MicroPather/micropather.h

```diff
@@ -349,8 +349,8 @@ namespace micropather
 		const Item* Find(void* start, void* end);
 
 		Item* mem = nullptr;
-		int allocated = 0;
-		int nItems = 0;
+		unsigned allocated = 0;

```

Comment:
Personally, I hate this style. Yes, I know it defaults to `unsigned int`, but ["Specifics, Bob"](https://www.youtube.com/watch?v=qHDSNs9wBpU)

Reply:
I hate that style too. I chose it to be consistent with the rest of the code in that file.

Reply:
Fair enough.